client/flutter/lib/pages/onboarding/onboarding_page.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:WHOFlutter/pages/onboarding/legal_landing_page.dart';
+import 'package:WHOFlutter/pages/onboarding/location_sharing_page.dart';
+import 'package:WHOFlutter/pages/onboarding/notifications_page.dart';
+import 'package:flutter/material.dart';
+
+class OnboardingPage extends StatelessWidget {
+  final PageController pageController = PageController();
+
+  @override
+  Widget build(BuildContext context) {
+    return PageView(
+      controller: this.pageController,
+      physics: NeverScrollableScrollPhysics(),
+      children: <Widget>[
+        LegalLandingPage(this.pageController),
+        NotificationsPage(this.pageController),
```

Comment:
This change isn't essential but I thought it's worth pointing out.
You could reduce code duplication by adding `VoidCallback`s to these Widgets that invoke a function in `OnboardingPage` to handles switching pages via `this.pageController.nextPage(duration: Duration(milliseconds: 500), curve: Curves.easeInOut),` 

This would also make it easier to switch the implementation from a `PageView` to a different widget if necessary.